README.md

```diff
@@ -156,8 +154,41 @@ Here is a screenshot of the map-structure in our project:<br><br>
 
 ## 🔨 Install
 
+Here's a manual for installing this project.
+
+### 📡 Install Node.js & npm
+
+To start off, install Node.js & npm. Node.js is a platform built on Chrome's JavaScript runtime for easily building fast and scalable network applications. Npm is a Node Package Manager, to install and manage packages.
+
+To install Node.js and npm, go to <a href="https://nodejs.org/en/">https://nodejs.org/en/</a>, and download the latest version of Node.js on your device. Make sure to select the npm-install option while installing Node.js.
+
+### 📍 Clone the GitHub repository
+
+To get the project on your code-editor, you have to clone it from our GitHub repository. (<a href="https://github.com/veerleprins/FemAlert-meesterproef">https://github.com/veerleprins/FemAlert-meesterproef</a>)
+<br>
+To do this, type the following command in your terminal:
+<br><br>
```

Comment:
All <br> 's can be removed. Instead, you can put 4 spaces on the line above and enter twice! Then you get the same effect 😄